### ISSUE TITLE
After Fugu14 setup & update, launched app screen is blank instead of "install untether"

### ISSUE THREAD

**Issue by mooseknuckleking** [LinusHenze/Fugu14#215]
Device info:
iPhone 11 pro max, ios 14.3
Root Filesystem Status: Clean, no jailbreak installed
Developer Apple ID

Issue:
After the Fugu14 app successfully opens and completes the "setup," and further seems to successfully update with the usage of python script, we receive the message: "Done! Open the Fugu14App again and follow the on-screen instructions!." But instead of the supposed "Install Untether" Screen we may expect, we get a blank white or black screen with only spotlight search available. The app may stay open or crash.

Methods used:
-Altstore has not been updated to auto recognize 14.3 devices yet, so this method was not used
-Used the ios_install.py script, followed directions, same result.
-Manually built the app in xcode, ran setup, injected update, still same issue. Untether not initiated as there is no button

Only errors spotted, during fugu14 update process:

>Installing Fugu14 update
>
>WARNING: could not locate iTunesMetadata.plist in archive!
>WARNING: could not locate Payload/Fugu14App.app/SC_Info/Fugu14App.sinf in archive!
>Copying 'tools/Fugu14_Pwn.ipa' to device... DONE.
>Installing 'de.linushenze.Fugu14App'
>Install: CreatingStagingDirectory (5%)
>Install: ExtractingPackage (15%)
>Install: InspectingPackage (20%)
>Install: TakingInstallLock (20%)
>Install: PreflightingApplication (30%)
>Install: InstallingEmbeddedProfile (30%)
>Install: VerifyingApplication (40%)
>Install: CreatingContainer (50%)
>Install: InstallingApplication (60%)
>Install: PostflightingApplication (70%)
>Install: SandboxingApplication (80%)
>Install: GeneratingApplicationMap (90%)
>Install: Complete
>
>
>Done! Open the Fugu14App again and follow the on-screen instructions!
>Once you are jailbroken, have rebooted and unlocked your device, connect to it by running the following commands:
   > iproxy 1337 1337  # Terminal Window A
   > nc localhost 1337 # Terminal Window B
>You should now see a prompt like this: 'iDownload>'. Type 'bash' to get a root shell
>Exit status: 0
>Saving session...
>...copying shared history...
>...saving history...truncating history files...
>...completed.
>
>[Process completed] 

-Does the Fugu14app.sinf contain the GUI for the updated app, and it is not located in the built archive therefore it's not being added in??
Wondering what the next steps are. If there is a manual way to perform the "install untether" option, such as running "runjailbreakd.js" using filza? Just attempting to turn this semi-tethered jailbreak untethered here.

**Comment by lafiean** [comment]
I have the Exact same issue but trying to install using AltStore

IPhone 11 Pro Max
IOS 14.5

**Comment by udevsharold** [comment]
You need to do them in non-jailbroken state.

**Comment by Midou36O** [comment]
> You need to do them in non-jailbroken state.

> Root Filesystem Status: Clean, no jailbreak installed

I don't think that's the problem..

**Comment by Midou36O** [comment]
Altough i did try the fugu14 jailbreak using altstore, everything goes flawlessly, then it gets stuck at the fugu14 black spotlight search

**Comment by DogeLord081** [comment]
It happens to me too

**Comment by devsecopsale** [comment]
same problem here. Did you find any solution?

**Comment by DogeLord081** [comment]
> same problem here. Did you find any solution?

Yep I did. All I had to do was reboot my phone and try again and it worked!

**Comment by Midou36O** [comment]
Didn't work for me, but i found a better solution using the perma-sign bug :)
